rq/worker.py

```diff
@@ -974,6 +977,12 @@ def handle_job_failure(self, job, queue, started_job_registry=None,
                                                         job_class=self.job_class, serializer=job.serializer)
                 failed_job_registry.add(job, ttl=job.failure_ttl,
                                         exc_string=exc_string, pipeline=pipeline)
+                try:
+                    pipeline.execute()
```

Comment:
Can we change this to use `with suppress(Exception)`? It will give us better test coverage since the `pass` line will not be tested.

Also, is there a specific exception that covers Redis connection errors? Just so that we don't accidentally swallow other classes of exceptions.

Reply:
Change completed. Logic now suppresses on redis connection errors with `with suppress(redis.exceptions.ConnectionError):`